src/frequenz/sdk/timeseries/_formula_engine/_formula_engine.py

```diff
@@ -411,10 +449,12 @@ class FormulaBuilder:
         following calls need to be made:
 
         ```python
-        channel = Broadcast[Sample]("channel")
+        from frequenz.sdk.timeseries._quantities import Quantity
```

Comment:
Better not to expose internal modules in examples.

Also, besides allowing instantiating a unit-less `Quantity` or not, I would use a concrete quantity with a unit in examples to avoid encouraging users to use `Quantity` directly, which should be pretty rare.

Same for all the other examples in this file.

```suggestion
        from frequenz.sdk.timeseries import Power
```

Reply:
done